src/pages/task/testsTable/TestsTableCore.tsx

```diff
@@ -45,38 +42,25 @@ export const TestsTableCore: React.FC = () => {
   const { replace } = useHistory();
   const { search, pathname } = useLocation();
 
-  // initial query variables to use when making first gql query
-  const [initialQueryVariables] = useState<TaskTestsQueryVariables>(
-    getQueryVariables(search, resourceId)
-  );
-  const { cat, dir } = initialQueryVariables;
+  const queryVariables = getQueryVariables(search, resourceId);
+  const { cat, dir, pageNum, limitNum } = queryVariables;
   const columns = useSetColumnDefaultSortOrder<TestResult>(
     columnsTemplate,
     cat,
     dir
   );
 
   // initial request for task tests
-  const { data, refetch, networkStatus } = useQuery<
+  const { data, loading, startPolling, stopPolling } = useQuery<
     TaskTestsQuery,
     TaskTestsQueryVariables
   >(GET_TASK_TESTS, {
-    variables: initialQueryVariables,
-    notifyOnNetworkStatusChange: true,
-    fetchPolicy: "network-only",
+    variables: queryVariables,
+    pollInterval,
   });
+  useNetworkStatus(startPolling, stopPolling);
 
-  useDisableTableSortersIfLoading(networkStatus);
-
-  // poll task tests
-  const isOffline = useNetworkStatus();
-  const { showSkeleton } = usePollQuery({
-    networkStatus,
-    getQueryVariables,
-    refetch,
-    search,
-    isOffline,
-  });
+  useDisableTableSortersIfLoading(loading);
```

Comment:
`loading` won't represent loading states from polled queries unless we set `notifyOnNetworkStatusChange: true` in the query options. We should add this setting to disable the sorters whenever a table request is in flight.

Reply:
I dont think we should do this. Polling happens in the back ground and if there is a change in the data it would update the UI to reflect that. If there is no change to the data the ui remains the same. 

Examining this hook i dont think this condition is ever hit/used. I slowed down the network request to watch what happens on the page. When we load new data a loading bar is displayed since its the first time its been requested. Which i think is fine since its in response to a user action. 
![page swap](https://user-images.githubusercontent.com/4605522/95106721-ddf5be00-0706-11eb-9ea9-5a14619fcab3.gif)

But when we poll in the background i don't think it should be noticeable to the user, We shouldn't indicate that to the user in anyway apart from updating the data on the screen if there are changes. `notifyOnNetworkStatusChange` Would visibly notify the user that something was loaded even tho nothing may have changed on screen.